sci-misc/boinc/boinc-7.2.44-r1.ebuild

```diff
@@ -99,7 +109,12 @@ src_install() {
 	keepdir /var/lib/${PN}
 
 	if use X; then
-		newicon "${S}"/packages/generic/sea/${PN}mgr.48x48.png ${PN}.png
+		# Create new icons. bug 593362
+		local SIZES="256 192 128 96 72 64 48 36 32 24 22 16"
+		for s in $SIZES ; do
+			convert "${DISTDIR}"/${PN}.tif -resize ${s}x${s} "${WORKDIR}"/boinc_${s}.png || die
+			newicon -s $s "${WORKDIR}"/boinc_${s}.png boinc.png
+		done
 		make_desktop_entry boincmgr "${PN}" "${PN}" "Math;Science" "Path=/var/lib/${PN}"
 	fi
 
```

Comment:
check the `pkg_preinst`: you're assigning to group and group**s**, is this right?